backend/src/models/user.js

```diff
@@ -17,7 +18,25 @@ const UserSchema  = new mongoose.Schema({
         type: String,
         required: true,
         unique: true
-    }
+    },
+    firstName: {
+        type: String,
+        required: true
+    },
+    surname: {
+        type: String,
+        required: true
+    },
+    phone: String,
+    picture: {
+        large: {data: Buffer, imageType: String},
```

Comment:
Are you sure about this? I really don't think this could be effective.

Reply:
https://stackoverflow.com/questions/3748/storing-images-in-db-yea-or-nay?page=1&tab=active#tab-top

The most relevant answer:

> I'm in charge of some applications that manage many TB of images. We've found that storing file paths in the database to be best.
> 
> There are a couple of issues:
> 
> database storage is usually more expensive than file system storage
>  - you can super-accelerate file system access with standard off the shelf products
>  - for example, many web servers use the operating system's sendfile() system call to asynchronously send a file directly from the file system to the network interface. Images stored in a database don't benefit from this optimization.
>  - things like web servers, etc, need no special coding or processing to access images in the file system
>  - databases win out where transactional integrity between the image and metadata are important.
>  - it is more complex to manage integrity between db metadata and file system data
>  - it is difficult (within the context of a web application) to guarantee data has been flushed to disk on the filesystem

In company where I worked, I had a large issue with this storing images into database. The problem was, that the search in the database was really slow, because there was images. Databases are really made for structured data, not for images. So I think, we should really change it to just string.

Reply:
Done